bfscripts.spec.rpkg

```diff
@@ -0,0 +1,56 @@
+%global debug_package %{nil}
+
+Summary: Useful scripts for managing Mellanox BlueField hardware.
+Name: mlxbf-bfscripts
+Version: 3.0.alpha3
+Release: 1%{?dist}
+License: BSD
+
+Source: {{{ git_dir_pack dir_name=bfscripts source_name=mlxbf-bfscripts.tar.gz }}}
+
+BuildRequires: python3-devel
+BuildRequires: /usr/bin/pathfix.py
+
+Requires: mlxbf-bootctl
+Requires: mlxbf-aarch64-firmware
```

Comment:
Any reason we called it aarch64 fw? Just wondering why not mlxbf-firmware directly. People might think we have firmware for each CPU architecture...

Reply:
I was concerned that if we call it mlxbf-firmware it might be confused with the connectX firmware. Fedora guidelines say that firmware of any kind needs to end in "-firmware", so I opted for the aarch64 tag (https://fedoraproject.org/wiki/Licensing:Main#Binary_Firmware).